### DISCUSSION TITLE
Welcome to the Discover Three.js Discussion Forum

### DISCUSSION THREAD

**Discussion by looeee** [looeee/discoverthreejs-site#8] [General]
## 👋 Welcome!
  All readers of Discover three.js are welcome here! 
  This is a place for you to:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and be open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself.

**Comment by ahsan-a** [comment]
Hello! I'm Ahsan, a 13 year old vue and typescript shill. I recently became interested in three.js and had no idea where to start learning it. I came here to say thanks for this site 😃

**Reply by looeee** [reply]
Hey Ahsan! Good to meet you 😁 

Make sure to stop by the forum and discord too:

https://discord.com/invite/jYMz4sX
https://discourse.threejs.org/
